packages/atomic/src/templates/default.html

```diff
@@ -59,6 +61,10 @@
   .template-container b {
     font-weight: var(--atomic-font-bold);
   }
+
+  atomic-result-printable-uri {
+    line-height: 20px;
+  }
```

Comment:
This style would better be placed inside `atomic-result-printable-uri` component, so that it is available by default.

It would also be better to specify a unit-less line-height so that it adapts to the font-size. A value of between `1.5` and `2` is typically a good starting point.

Ref: https://www.w3schools.com/cssref/pr_dim_line-height.asp

Reply:
I can change to a unit, sure. 

And no, I don't think it should be embedded inside the component. A "line height that looks good" is dependent on the context of the page/the layout etc.

Reply:
I agree the exact number will depend on the font, designer's preference, layout etc. but I also think the browser default of `1.2` is too close.

There are many articles about this on the web. If you inspect this paragraph, Github is using a line-height `1.5` for example. This could be a nice default potentially for all of atomic, not just the printable-uri.

Some references:
- [Article](https://www.google.com/search?q=line-height+css+best+practice&rlz=1C1CHBD_enCA926CA926&ei=_XWJYOnXBcGq1QGjnIz4BA&oq=line+height+css+bes&gs_lcp=Cgdnd3Mtd2l6EAMYADIGCAAQFhAeMgYIABAWEB46BwgAEEcQsAM6BAgAEEM6BQgAEJECOgYIABAHEB46AggAOggIIRAWEB0QHlD9XlixiQFgxpABaAxwAngAgAHVAogBwBCSAQc5LjQuMi4xmAEAoAEBqgEHZ3dzLXdpesgBCMABAQ&sclient=gws-wiz) used by Google's Q&A suggesting `1.5`.
- [Article](https://www.justinmind.com/blog/best-ux-practices-for-line-spacing/) suggesting `1.4`.

Reply:
Fair enough !